confd/templates/production.tmpl

```diff
@@ -173,6 +174,7 @@
   },
   "mapFacadeConfig": {
   	"showOne2One": {{ getv "/ansyn/show/one2one" "true" }},
+	"timeToDisplayButtonToast": 10000,
```

Comment:
this is not related that much to the mapFacadeConfig direclty,
maybe create seperate config for that?
חחחחחחחחחחחח מצחיק וואו אלאור שם הערה על הקונפיג חחחחחחחחחחחחחחחחחחח